test/inductor/test_torchinductor.py

```diff
@@ -10779,6 +10779,22 @@ def fn(x, x2):
         _, code = run_and_get_code(fn, x, x2)
         FileCheck().check("aten.view.dtype(reinterpret_tensor").run(code[0])
 
+    @requires_gpu()
+    def test_scalar_cpu_tensor_arg(self):
+        def fn(x, y):
+            return x + y.sum()
+
+        test_dtypes = [
+            torch.float32,
+            torch.float64,
+            torch.float16,
+            torch.bfloat16,
```

Comment:
Not sure why it fails for bfloat16 and both x,y are on GPU. This case doesn't utilize this PR's code.

Reply:
The error is the unit test config problem. fixed in https://github.com/pytorch/pytorch/pull/132702/commits/502ed996db5172e5bfec8851d5f3d1bd2681a395